src/content/workers/smelter.yaml

```diff
@@ -6,8 +6,13 @@ type: gatherer
 traits:
   primary:
     name: Athletics
-    effect: Faster smelting
+    effect: Amount of furnaces that can be used
```

Comment:
Please once again use an override, for this one, you can combine it with the 1.19 one

Reply:
Oh! D'oh! Yeah, of course